src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -475,6 +496,7 @@ ScriptEditorWidgetColorOptions::ScriptEditorWidgetColorOptions(QWidget * pParent
 	gbox->setInsideSpacing(0);
 
 	addColorSelector(gbox, __tr2qs_ctx("Background:", "editor"), &g_clrBackground, true);
+	addColorSelector(gbox, __tr2qs_ctx("Background Highlight:", "editor"), &g_clrBackgroundHighlight, true);
```

Comment:
wouldnt it be nicer if this was simply called **Highlight current line** with no Camel Case since it is a label?